exchanges/account/account.go

```diff
@@ -92,16 +95,12 @@ func GetHoldings(exch string, creds *Credentials, assetType asset.Item) (Holding
 	defer service.mu.Unlock()
 	accounts, ok := service.exchangeAccounts[exch]
 	if !ok {
-		return Holdings{}, fmt.Errorf("%s %s %w", exch, assetType, errExchangeHoldingsNotFound)
+		return Holdings{}, fmt.Errorf("%s %s %w", exch, assetType, ErrExchangeHoldingsNotFound)
```

Comment:
Nit:
* Encase the error value so it's clear if it held something empty-looking.
```suggestion
		return Holdings{}, fmt.Errorf("%s %w: `%s`", exch, ErrExchangeHoldingsNotFound, assetType)
```